### DISCUSSION TITLE
Cannot read property setAcessToken of undefined

### DISCUSSION THREAD

**Discussion by ghost** [rnmapbox/maps#2364] [Q&A]
Hi folks, I am trying to integrate Mapbox to my project but hitting an error in the line that calls for the token MapboxGL.setAccessToken.
The error reads "Cannot read property setAcessToken of undefined.
I have combed through your docs with care and to no avail I have followed the lead of those who have encountered similar problems.
Is it possible that something broke after PR [#2343](https://github.com/rnmapbox/maps/pull/2343) was merged?
If not, please help, I'm playing with React Native 0.69.0, Xcode 14.0.1, iOS 12.6, and CocoaPods 1.11.3

**Comment by mfazekas** [comment]
Check that your Xcode project has MGLModule, and place a breakpoint in `constantsToExport` say.

**Reply by ghost** [reply]
Thank you very much for your prompt response. I must confess that after I posted this I decided to start from scratch and all went well, so I am sorry to say that now we'll have no idea what was the source of the error. However, I did have an issue with Ruby versions during the installation so most probably something got messed up while I was solving that problem. On the other hand, the good news is that this time the installation was smooth. 
This is a fantastic tool and community, I'm looking forward to participating.
